data/hydraulic-data.py

```diff
@@ -0,0 +1,147 @@
+
+                # import all data of volume flow sensors
+                df_vf1 = import_data_field(zip, "FS1.txt", "fsa_vol_flow")
+                df_vf2 = import_data_field(zip, "FS2.txt", "fsb_vol_flow")
+
+                # import all data of temperature sensors
+                df_ts1 = import_data_field(zip, "TS1.txt", "tsa_temp")
+                df_ts2 = import_data_field(zip, "TS2.txt", "tsb_temp")
+                df_ts3 = import_data_field(zip, "TS3.txt", "tsc_temp")
+                df_ts4 = import_data_field(zip, "TS4.txt", "tsd_temp")
+
+                # import data of pump efficiency sensor
+                df_pe1 = import_data_field(zip, "EPS1.txt", "pump_eff")
+
+                # import data of vibrations sensor
+                df_vs1 = import_data_field(zip, "VS1.txt", "vs_vib")
+
+                # import data of cooling efficiency sensor
+                df_ce1 = import_data_field(zip, "CE.txt", "cool_eff_pct")
+
+                # import data of cooling power sensor
+                df_cp1 = import_data_field(zip, "CP.txt", "cool_pwr_pct")
+
+                # import data of efficiency factor sensor
+                df_ef1 = import_data_field(zip, "SE.txt", "eff_fact_pct")
+
+                # import one of the hydraulic condition monitoring labels from profile based on target label
+                df_hyd_cond_monitor_label_y = import_target_label(zip, "profile.txt", target_label)        
+                
+                # Combine all sensor Dataframes and then add hydraulic condition monitoring label as last column
+                df_hyd_cond_monitor_data_x = pd.concat([df_ps1, df_ps2, df_ps3, df_ps4, df_ps5, df_ps6, df_vf1,
+                                                df_vf2, df_ts1, df_ts2, df_ts3, df_ts4, df_pe1, df_vs1,
+                                                df_ce1, df_cp1, df_ef1, df_hyd_cond_monitor_label_y], axis=1)
+                dt.Frame(df_hyd_cond_monitor_data_x).to_csv(output_file) 
+
+        temp_path = os.path.join(h2oaicore.systemutils.config.data_directory,
+                                 h2oaicore.systemutils.config.contrib_relative_directory,
+                                 "hydraulic")
+        os.makedirs(temp_path, exist_ok=True)
+
+        # url contains hydraulic systems condition monitoring data set
+        link = "https://archive.ics.uci.edu/ml/machine-learning-databases/00447/data.zip"
+        # returns temp_path to filename with filename extracted from URL
+        file = h2oaicore.systemutils_more.download(link, dest_path=temp_path)
+        # Choose a target_label: cool_cond_y, valve_cond_y, pump_leak_y, acc_gas_leak_y, stable_y
+        # The target_label you choose will get added on as the last column in the training dataset
+        target_label = "cool_cond_y"
+        # creates csv destination filepath to store processed hydraulic condition monitoring data
+        output_file = os.path.join(temp_path, "hydCondMonitorData.csv")
```

Comment:
qq/nit: Is there a significance for the filename being `camelCase` instead of `under_scored`?  It _might_ be beneficial to use underscores as it's more typical in filesystem naming and also matches python variable naming.

Just curious, and sharing thoughts, I don't mean to push.